AI_VoiceOver/Options.lua

```diff
@@ -184,10 +184,7 @@ local GeneralTab =
                     type = "toggle",
                     order = 7,
                     name = "Hide NPC Portrait",
-                    desc = "Talking NPC portrait will not appear when voice over audio is played.\n\n" ..
-                            Utils:ColorizeText("This might be useful when using other addons that replace the dialog experience, such as " ..
-                                Utils:ColorizeText("Immersion", NORMAL_FONT_COLOR_CODE) .. ".",
-                                GRAY_FONT_COLOR_CODE),
+                    desc = "Talking NPC portrait will not appear when voice over audio is played.\n\n",
```

Comment:
Why remove this part, has is stopped being relevant to Immersion?
And if you're removing it, remove the newlines at the end too.

Reply:
I was under the impression that it's only the animated portrait that was an issue with Immersion, because Immersion already provides its own animated portrait. I don't see why it's beneficial to hide the entire frame and thus have no way to see the queue and to pause/resume the sound apart from the minimap button.